qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1520A.py

```diff
@@ -89,6 +92,22 @@ def _set_voltage_ac(self, value: float) -> None:
 
         self.write(msg.message)
 
+    def _get_dcv(self):
+        if self.is_enabled():
+            msg = MessageBuilder().lrn_query(self.channels[0])
+            response = self.ask(msg.message)
+            d = parse_dcv_measurement_response(response)
+        return d or None
+
+    def _get_voltage_dc(self) -> Optional[float]:
+        return float(self._get_dcv()['voltage_dc'])
```

Comment:
```suggestion
    def _get_voltage_dc(self) -> Optional[float]:
        dcv = self._get_dcv()
        return float(dcv['voltage_dc']) if dcv is not None else None
```

Reply:
Yes either that or remove the Optional from the type